Gemfile

```diff
@@ -20,6 +20,11 @@ gem 'uri-js-rails' # URI manipulation
 gem 'bootstrap-datepicker-rails'
 gem 'paperclip', '~> 5.2'
 
+gem 'turbolinks', '~> 5.0.0'
```

Comment:
Is turbolinks really required? We removed this for good reasons, unfortunately, and I don't trust that adding it back will be consequences-free.

Reply:
I'll remove it an see if we can go without